app/lib/actions/katello/repository/upload_package_group.rb

```diff
@@ -0,0 +1,34 @@
+module Actions
+  module Katello
+    module Repository
+      class UploadPackageGroup < Actions::EntryAction
+        def plan(repository, params)
+          action_subject(repository)
+          unit_key = {"repo_id": repository.pulp_id, "id": params[:name].parameterize.underscore}
+          metadata = {"unit_metadata": params}
```

Comment:
this ends up passing {unit_metadata: unit_metadata: {parmas}}    to pulp, no need to wrap it here.